ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -313,6 +313,20 @@ <h3>Source database details:</h3>
         Note: Spanner migration tool is creating datastream and dataflow resources. Please look at the terminal logs to
         check the progress of resource creation. All created resources will be displayed here once they are generated.
     </div>
+    <div *ngIf="isResourceGenerated && resourcesGenerated.MonitoringDashboardName!==''">
```

Comment:
Will MonitoringDashboardName not be null in case of sharded migration? Since aggregated dashboard is not a part of current PR, I am a bit confused

Reply:
Following the current convention we always set this value even for sharded migrations: https://github.com/GoogleCloudPlatform/spanner-migration-tool/blob/master/streaming/streaming.go#L721-L725